target/linux/ath79/image/generic.mk

```diff
@@ -314,6 +331,24 @@ define Device/avm_fritzdvbc
 endef
 TARGET_DEVICES += avm_fritzdvbc
 
+define Device/belkin_f9j1108-v2
+  SOC := qca9558
+  DEVICE_VENDOR := Belkin
+  DEVICE_MODEL := F9J1108v2 (AC1750 DB Wi-Fi)
+  DEVICE_PACKAGES += ath10k-firmware-qca988x-ct kmod-usb2 kmod-usb3 kmod-usb-ledtrig-usbport
+  LOADER_TYPE := bin
+  LOADER_FLASH_OFFS := 0x50000
+  COMPILE := loader-$(1).bin loader-$(1).uImage
+  COMPILE/loader-$(1).bin := loader-okli-compile
+  COMPILE/loader-$(1).uImage := append-loader-okli $(1) | pad-to 64k | lzma | \
+	uImage lzma
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma -M 0x4f4b4c49
+  IMAGE/factory.bin := edimax-headers $(1) F9J1108v1 BR-6679BAC | append-kernel | append-rootfs | check-size
+  IMAGES += factory.bin
+  IMAGE_SIZE := 14464k
```

Comment:
for consistency with the rest of the makefile

put KERNEL line before LOADER_TYPE

reorder the other three as
IMAGE_SIZE
IMAGES
IMAGE/factory.bin